README.md

```diff
@@ -7,14 +7,17 @@
 
 ![Build Status - GitHub][actions-build-status]
 [![PyPI Version][pypi-img]][pypi-url]
-[![Coverage Status][codecov-img]][codecov-url]
+![GitHub Release Date][github-release-date]
+[![GitHub commits latest](https://img.shields.io/github/commits-since/Clinical-Genomics/scout/latest)](https://GitHub.com/Clinical-Genomics/scout/commit/)
+[![GitHub commit rate](https://img.shields.io/github/commit-activity/w/Clinical-Genomics/scout)](https://GitHub.com/Clinical-Genomics/scout/pulse/)
+[![GitHub commits](https://img.shields.io/github/commits-since/Clinical-Genomics/scout/v4.0.0.svg)](https://GitHub.com/Clinical-Genomics/scout/commit/)
```

Comment:
Ah look, this one didn't work: 
<img width="195" alt="image" src="https://user-images.githubusercontent.com/28093618/203574854-96b920b8-f64e-47d3-ad4f-394347567166.png">

Reply:
Ah no, it the other one. Never mind!

Reply:
Ah, but that is latest; the slightly large one is there as well:
![Screenshot 2022-11-23 at 15 43 17](https://user-images.githubusercontent.com/758570/203575306-328055db-38f5-42ab-8738-2f07a1ced26f.png)